### PR TITLE
Discord button

### PR DESCRIPTION
**Pull request by Sprtcrnbry**
Added Discord invite link to website

Signed-off-by: Sprtcrnbry <sprtcrnbry@gmail.com>

### CODE REVIEW

**Comment by ammarfaizi2** [comment]
I am going to update the timeline history, will merge together with this PR after finishing it. So I will commit to your branch soon.

**Comment by Sprtcrnbry** [comment]
@ammarfaizi2 Alright, thanks

File: public/index.html [hunk 1]
```diff
@@ -63,6 +63,9 @@ <h1 class="title display-2 fw-bolder">GNU/Weeb</h1>
               <a class="gw-button" target="_blank" href="//git.io/GNUWeeb">
```
Comment (siarie) [inline public/index.html]: @Sprtcrnbry can you change the link (telegram and Github) to `https://` too?
I previously used `//` instead of `https: //`?

**Review by siarie — COMMENTED** [review]
Which one is better?
`https://` or `//`

File: public/index.html (hunk 1 above)
Comment (ammarfaizi2) [reply public/index.html]: I am addressing your review. Please wait for it.

Comment (Sprtcrnbry) [reply public/index.html]: im not sure whats the difference, but im sure by using `https://` it forces https instead (more secure).

Comment (ammarfaizi2) [reply public/index.html]: @Sprtcrnbry using // will adapt to the current page scheme, if it uses HTTPS then it be HTTPS, the same happens with HTTP.

And yes, using https:// forces it to be HTTPS URL regardless accessed scheme.

Comment (ammarfaizi2) [reply public/index.html]: I am resolving this review.

**Comment by ammarfaizi2** [comment]
Any further review?

**Review by siarie — APPROVED** [review]
LGTM

**Review by ammarfaizi2 — APPROVED** [review]

**Pull request merged**